target/linux/generic/files/drivers/net/phy/rtl8367b.c

```diff
@@ -962,6 +1003,17 @@ static int rtl8367b_setup(struct rtl8366_smi *smi)
 			RTL8367B_PORT_MISC_CFG_EGRESS_MODE_ORIGINAL <<
 				RTL8367B_PORT_MISC_CFG_EGRESS_MODE_SHIFT);
 
+	/*
+	 * Enable each phy port.
+	 */
+	for (i = 0; i < 5; i++) {
+		int data;
+		rtl8367b_read_phy_reg(smi, i, 0, &data);
+		data &= 0xF7FF;
+		data |= 0x200;
+		rtl8367b_write_phy_reg(smi, i, 0, data);
+	}
```

Comment:
Why didn't we need this code before?

Reply:
I guess it's already done by the bootloader. Unless they get reseted somehow we should be careful to not break other chips' support and test it without these lines.